### PR TITLE
Automerge renovate PRs

### PR DESCRIPTION
**Pull request by luhi-DT**
## Description

Automerge renovate PRs

## How can this be tested?

Not needed

### CODE REVIEW

**Review by andriisoldatenko — DISMISSED** [review]

File: .github/renovate.json5 [hunk 1]
```diff
@@ -60,6 +60,7 @@
         "minor",
       ],
       enabled: false,
+      automerge: true,
```
Comment (chrismuellner) [inline .github/renovate.json5:63]: do you want to automerge for both `main` and release branches?

**Review by albertogdd — DISMISSED** [review]
image

File: .github/renovate.json5 (hunk 1 above)
Comment (luhi-DT) [reply .github/renovate.json5:63]: you are right, i need to enable automerge for all package rules, and yes i would also auto-merge release branches

**Review by andriisoldatenko — APPROVED** [review]

**Review by albertogdd — APPROVED** [review]

**Pull request merged**